packages/amplication-client/src/Components/ComplteSignupButton.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useCallback, useState } from "react";
+import {
+  Button,
+  Dialog,
+  EnumTextColor,
+  JumboButton,
+} from "@amplication/ui/design-system";
+
+import { useTracking } from "../util/analytics";
+import { AnalyticsEventNames } from "../util/analytics-events.types";
+
+import { CompleteSignupDialog } from "./CompleteSignupDialog";
+import { useMutation } from "@apollo/client";
+import { COMPLETE_SIGNUP_WITH_BUSINESS_EMAIL } from "../User/UserQueries";
+import { CommitBtnType } from "../VersionControl/Commit";
+
+export enum EnumButtonLocation {
+  Project = "Project",
+  Resource = "Resource",
+  EntityList = "EntityList",
+  SchemaUpload = "SchemaUpload",
+  PreviewBtm = "PreviewBtm",
+  Architecture = "Architecture",
+}
+
+type Props = {
+  buttonText?: string;
+  // autoRedirectAfterClick?: boolean;
+  buttonType?: CommitBtnType;
+};
+
+export const CompleteSignupButton: React.FC<Props> = ({
```

Comment:
Move the file to ./User together with the rest of the components related to the preview signup flow